### ISSUE TITLE
tags: --tags option not properly picking up tests (all scenarios skipped) when using logical "or"

### ISSUE THREAD

**Issue by glenn-barker** [behave/behave#763]
I have the following (very) simple feature file:

```
Feature: FizzBuzz
  As a tester
  I want to be able to run both "Fizz" or "Buzz" scenario by specifying their appropriate tags
  So that I can confirm granular control over scenario execution


@fizz
Scenario: Fizz scenario
  When this scenario is executed
  Then "fizz" should be printed to the console


@buzz
Scenario: Buzz scenario
  When this scenario is executed
  Then "buzz" should be printed to the console
```

I'm able to run the scenarios on an individual basis by running:
`$ behave --tags="fizz"`
`$ behave --tags="buzz"`
Both of these produce the expected output: 1 scenario executed, 1 skipped.

However, consider the following:
`$ behave --tags="fizz or buzz"`

Expected behavior: 2 scenarios executed, 0 skipped.
Actual behavior: 0 scenarios executed, 2 skipped.

Basically, trying to use "or" to select multiple tags to execute on just results in every scenario being skipped.

REFERENCE: https://behave.readthedocs.io/en/latest/behave.html#tag-expression

**Comment by jenisys** [comment]
Which version of `behave` are you using ?
Any version below `1.2.7` does not support the new tag-expression v2 (as described in the docs).

SEE ALSO:

* https://behave.readthedocs.io/en/latest/new_and_noteworthy_v1.2.7.html

**Comment by glenn-barker** [comment]
I see, I'm on `1.2.6` so that explains it. That seems to be the latest stable version that you get when you install via `pip install behave`, so is `1.2.7` still considered a bleeding-edge / unstable version?

I guess I was just looking at the wrong docs, so I'm closing this.

**Comment by vishrantgupta** [comment]
@jenisys it's been a long time since a stable version of `behave` has been released, any plans of releasing the stable version of behave which can support https://behave.readthedocs.io/en/latest/tag_expressions.html#tag-matching-with-tag-expressions?

**Comment by vishrantgupta** [comment]
Also, how can I install behave 1.2.7 any PyPi indexUrl?

**Comment by bittner** [comment]
> @jenisys it's been a long time since a stable version of `behave` has been released, any plans of releasing the stable version of behave which can support https://behave.readthedocs.io/en/latest/tag_expressions.html#tag-matching-with-tag-expressions?

You may take a look at the discussion in #855 an #1059. How to install any code point from Git using pip is also mentioned in the latter (and in the docs).

**Comment by jenisys** [comment]
https://behave.readthedocs.io/en/latest/install.html#using-the-github-repository
